cmd/statusd/main.go

```diff
@@ -251,7 +257,12 @@ func makeNodeConfig() (*params.NodeConfig, error) {
 		nodeConfig.ClusterConfig.BootNodes = nil
 	}
 
-	nodeConfig.NoDiscovery = !(*discovery)
+	nodeConfig.ClusterConfig.RendezvousNodes = []string(rendezvous)
+	nodeConfig.EnabledDiscoveries = []string(dtypes)
+	if len(nodeConfig.EnabledDiscoveries) == 0 {
```

Comment:
Not sure if this is needed. It might be but I guess only if `* discoveryFlag` is `true`.

Reply:
i added it to preserve original app behaviour